packages/react/src/analytics/integrations/Guardian/Guardian.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Guardian Integration. Loads the Guardian script and calls its methods.
+ *
+ * @example <caption>Adding Guardian integration to analytics</caption>
```

Comment:
This description is not correct since this integration does not load a script as Vitorino once did.